Oliver-ke/Teamwork
# #28594892 Setup test with mocha and chai

#### What does this PR do?
- add testing functionality

#### Description of Task to be completed?
- add mocha and chai to packages
- configure mocha for testing
- add basic test file

#### How should this be manually tested?
- clone this repository
- run `npm install` to get all dependencies
- run `npm run test` to run the test script
- you should see the test running and the test result

#### Any background context you want to provide?
none

#### What are the relevant project board stories?
[#28594892](https://github.com/Oliver-ke/Teamwork/projects/1#card-28594892)

#### Screenshots (if appropriate)
- none

#### Questions:
- none
